libs/ui/src/lib/circle/circle.ts

```diff
@@ -1,10 +1,11 @@
-export type CircleTheme = 'dark' | 'light';
+export type CircleTheme = 'dark' | 'light' | 'dark-filled';
```

Comment:
Just a thought, that instead of new theme 'dark-filled', this can also be achieved with
theme = 'light'
severity = 'info'
and adding a style attribute to the `cvi-ng-circle` component with the custom border colour (same as 'info' background) `style="--cvi-circle-border-color: --cvi-color-sea-green-10"`

Reply:
Good idea! Implemented!